eng/targetingpacks.targets

```diff
@@ -87,6 +87,16 @@
     </PropertyGroup>
   </Target>
 
+  <!-- SDK tries to download runtime packs when RuntimeIdentifier is set, remove them from PackageDownload item. -->
+  <Target Name="RemoveRuntimePackFromDownloadItem"
+          Condition="'$(_UseLocalTargetingRuntimePack)' == 'true' and '$(RuntimeIdentifier)' != ''"
+          AfterTargets="ProcessFrameworkReferences">
+    <ItemGroup>
+      <PackageDownload Remove="@(PackageDownload)"
+                       Condition="$([System.String]::Copy('%(Identity)').StartsWith('Microsoft.NETCore.App.Runtime'))" />
```

Comment:
```suggestion
                       Condition="$([System.String]::Copy('%(Identity)').StartsWith('$(SharedFrameworkName).Runtime'))" />
```

Reply:
You are missing part of my suggestion here :D You could just click on Commit suggestion ;)